src/api/solr-client.js

```diff
@@ -42,20 +42,18 @@ class SolrClient {
     const searchFieldsToMerge = queryToMerge.searchFields || [];
     const sortFieldsToMerge = queryToMerge.sortFields || [];
 
-    this.state.query.searchFields = this.state.query.searchFields
-      .map((sf) => searchFieldsToMerge.map((sfm) => sfm.field).indexOf(sf.field) > -1
-        ? {...sf, value: searchFieldsToMerge.find((sfm) => sfm.field === sf.field).value}
-        : sf);
+    // update all searchFields parameters
+    this.state.query.searchFields = searchFieldsToMerge
```

Comment:
Are we sure we aren't introducing bugs here?  We're overwriting more fields now, right?  Are we sure we want all of them?  I'm a little worried since they clearly went out of their way in the original code to not do this.

Reply:
Good points. How about following the original code and updating `hidden` and `collapse` with the same style?

Reply:
I'd feel better about that, so we'd know we weren't introducing unexpected new behavior.